main.go

```diff
@@ -19,16 +19,21 @@ func main() {
 	wps := mustGetEnvInt("WPS")
 	batchSize := mustGetEnvInt("BATCH_SIZE")
 	destination := mustGetEnvString("DESTINATION")
-	num_docs := getEnvDefaultInt("NUM_DOCS", -1)
+	numDocs := getEnvDefaultInt("NUM_DOCS", -1)
 	mode := getEnvDefault("MODE", "add")
-	patch_mode := getEnvDefault("PATCH_MODE", "replace")
+	idMode := getEnvDefault("ID_MODE", "uuid")
+	patchMode := getEnvDefault("PATCH_MODE", "replace")
+	collectMetrics := getEnvDefaultBool("METRICS", false)
 
-	if !(patch_mode == "replace" || patch_mode == "add") {
+	if !(patchMode == "replace" || patchMode == "add") {
 		panic("Invalid patch mode specified, expecting either 'replace' or 'add'")
 	}
 	if !(mode == "add" || mode == "patch") {
 		panic("Invalid mode specified, expecting 'add' or 'patch'")
 	}
+	if !(idMode == "uuid" || idMode == "sequential") {
```

Comment:
Currently patch mode required sequential inserts so that it doesn't need to hold all the _ids in memory while patching. It might be nice to add a check here for that.